mmcv/ops/nms.py

```diff
@@ -14,8 +15,9 @@
 class NMSop(torch.autograd.Function):
 
     @staticmethod
-    def forward(ctx, bboxes, scores, iou_threshold, offset, score_threshold,
-                max_num):
+    def forward(ctx: Any, bboxes: torch.Tensor, scores: torch.Tensor,
```

Comment:
Maybe we can use ``from torch import Tensor``,  and use `Tensor` to replace `torch.Tensor` in Type hint.